ts/src/core/query_impl.ts

```diff
@@ -9,6 +9,7 @@ export interface OrderByOption {
   // needed to know if we create a cursor based on this column to conver to timestamp and ISO string for
   // comparison
   // maybe eventually want a more generic version of this but for now this suffices
+  // @deprecated You no longer need to specify if a column is a date column
```

Comment:
you can just kill it. and we'll add this as a breaking 2.0 change that i keep forgetting to do